### DISCUSSION TITLE
How do I add or invite users?

### DISCUSSION THREAD

**Discussion by TechMechanik** [TandoorRecipes/recipes#2787] [Q&A]
Looking at the Django admin page, there isn't a way to add users, only change.
From the Spaces area, I can enter a token, but I don't have any way to send an invite.

Installation is on an Unraid server.

I feel like I am missing something in the setup, but after going through the documentation, I can't see anything that is standing out.

![Screenshot 2023-12-07 180141](https://github.com/TandoorRecipes/recipes/assets/1356227/f69a9e38-64a2-4345-97c7-2ef6c7350881)
![Screenshot 2023-12-07 180216](https://github.com/TandoorRecipes/recipes/assets/1356227/95085320-98a7-47aa-95f8-d07c39d36186)

**Comment by smilerz** [comment]
Space settings.

**Comment by dawierha** [comment]
[how to add user](https://docs.tandoor.dev/faq/#how-can-i-create-users)
